linera-core/src/client.rs

```diff
@@ -1773,6 +1785,13 @@ where
                     executed_block.message_id_for_operation(0, OPEN_CHAIN_MESSAGE_INDEX)
                 })
                 .ok_or_else(|| ChainClientError::InternalError("Failed to create new chain"))?;
+            // Add the new chain to the list of tracked chains
+            self.node_client
+                .track_chain(ChainId::child(message_id))
+                .await;
```

Comment:
Will that make the faucet track all the chains it opens?

Reply:
I think that's already the case, right?

Reply:
Probably, yes. Maybe also something to optimize away at some point.